app/views/posts/show.html.erb

```diff
@@ -7,11 +7,22 @@
             <span>comments: <%= @post.comments.count %>,</span>
             <span>likes: <%= @post.likes.count %></span>
         </div>
-        <p><%= @post.text %></p>
+        <p><%= @post.title %></p>
        
         <% @post.comments.each do |comment| %>
             <span class='commentor'><%= comment.author.name %>: </span>
             <span><%= comment.text %></span><br>
         <% end %>
     </div>
-    <% index = index + 1 %>
\ No newline at end of file
+<% index = index + 1 %>
+
+<%= form_with(model: @comment, url: "/users/#{@post.author.id}/posts/#{@post.id}/comments", method: :post, scope: :comment) do |form| %>
+  <div class="form-group">
+    <%= form.label :text %>
+    <%= form.text_area :text, class: 'form-control' %>
+  </div>
+
+  <%= form.submit 'Create Comment', class: 'btn btn-primary' %>
+<% end %>
```

Comment:
- [ ] Please add a form in the `views/comments/new.html.erb` file which accepts input for creating comments.

Reply:
But this was not specified in the requirements

Reply:
> But this was not specified in the requirements

Hi @Linushaddai99, 
I hope you're doing well,
It's required to create a `new` method in the comments controller and add a form here `views/comments/new.html.erb` and also a route to it because this is the correct way to do it,
and it's mentioned the code reviewer guidelines I have, So it's required for me to check for these changes in your code.